### PR TITLE
SU002-Adding-Boiler-Plate

### PR DESCRIPTION
**Pull request by Criofann**
- UI skeleton created using Node.js, Express and Nunjucks.
- Added the .gitignore file to protect sensitive information.
- Added the node.js.yml file to install all the dependancies

Screenshot 2023-12-01 at 11 39 36

Screenshot 2023-12-01 at 11 40 26

### CODE REVIEW

**Review by Joseph-Magee — APPROVED** [review]
looks fine

File: .gitignore
Comment (jkelso662) [inline .gitignore:1]: Not a blocker but would caution other folders like visual studio codes `code` folder and intellijs `idea` folders eta.

**Review by jkelso662 — APPROVED** [review]
Good description on the ticket. Seems good to me.

**Review by Joseph-Magee — APPROVED** [review]
All looks good to me

**Pull request merged**